internal/shared/semconv/env.go.tmpl

```diff
@@ -4,7 +4,7 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-package semconv
+package semconv // import "{{.pkg}}/internal/semconv"
```

Comment:
This shouldn't be necessary. The `make generate` command runs `vanity-import-fix`, which adds those automatically.